Epsilon.Host.WebApi/Controllers/ComponentController.cs

```diff
@@ -22,4 +24,12 @@ public async Task<ActionResult<CompetenceProfile>> GetCompetenceProfile()
 
         return competenceProfile;
     }
+    
+    [HttpGet("kpi_matrix")]
```

Comment:
At the moment this is something that is not needed as an API endpoint

Reply:
It is